boards/arm/thingy53_nrf5340/Kconfig

```diff
@@ -59,6 +59,24 @@ config DOMAIN_CPUNET_BOARD
 	  another board. For example hci_rpmsg on the nRF5340_cpunet for
 	  Bluetooth applications.
 
+config BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+	bool "MCUmgr and OTA DFU"
+	select MCUMGR
+	select MCUMGR_SMP_BT
+	select MCUMGR_CMD_IMG_MGMT
+	select MCUMGR_CMD_OS_MGMT
+	select NORDIC_QSPI_NOR
```

Comment:
Can we also by default enable: `CONFIG_IMG_ERASE_PROGRESSIVELY`?
Consider also aligning QSPI flash configuration to defaults that were used by Thigny:53:
```
CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096                                                   
CONFIG_NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE=16 
```

Reply:
Fixed